src/test/java/org/jenkinsci/plugins/github/test/GHMockRule.java

```diff
@@ -13,6 +12,7 @@
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
+import wiremock.org.eclipse.jetty.http.HttpStatus;
```

Comment:
NIT: generally I would not rely on on internal implementations, but use `java.net.HttpURLConnection.HTTP_CREATED`, but this was existing anyway.